metricbeat/docs/modules/sql.asciidoc

```diff
@@ -0,0 +1,43 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+
+[[metricbeat-module-sql]]
+[role="xpack"]
+== sql module
+
+beta[]
+
+This is the sql module that fetches metrics from a SQL database. You can define driver, datasource and SQL query.
+
+
+
+
+[float]
+=== Example configuration
+
+The sql module supports the standard configuration options that are described
+in <<configuration-metricbeat>>. Here is an example configuration:
+
+[source,yaml]
+----
+metricbeat.modules:
+- module: sql
+  metricsets:
+    - query
+  period: 10s
+  hosts: ["localhost"]
+
+  driver: "postgres"
+  datasource: "user=myuser password=mypassword dbname=mydb sslmode=disable"
+  sql_query: "select now()"----
```

Comment:
I think that docs builds are failing because of this.
```suggestion
  sql_query: "select now()"
  ----
```
To fix it please add a new line at the end of `x-pack/metricbeat/module/sql/_meta/config.yml`.

Reply:
Sorry, I changed the wrong fields.yml :|